proposals/0296-increase-tx-size.md

```diff
@@ -0,0 +1,243 @@
+Ixs [(u8, u8, u16)] -- Number matches NumInstructions. Values are 
+ (program_account_index, num_accounts, num_data_bytes)
+[TRAILING DATA SECTION] -- Length must be consistent with the sum of 
+ num_accounts and num_data_bytes in `Ixs`
+Signatures [[u8; 64]] -- Length of `num_required_signatures` from
+ `LegacyHeader`
+```
+
+The [TRAILING DATA SECTION] contains instruction's serialized account indexes
+and instruction data bytes,
+in the order they appear in the transaction. Each instruction's account indexes
+must appear before its data bytes.
+For example, if there are two instructions:
+
+1. `Instruction { account_indexes: [0, 1], data: [2, 3, 4] }`
+2. `Instruction { account_indexes: [5, 6], data: [7, 8] }`
+
+The TRAILING DATA SECTION must look like this:
+
+```
+[0, 1, 2, 3, 4, 5, 6, 7, 8]
+```
+
+Lengths are serialized in the earlier `Ixs` section, and so are not repeated in
+the trailing data section.
+
+This new `v1` transaction format notably does not include address lookup
+tables.
+
+With the new transaction size limit, it is possible to go past some other 
+implicit limitations that are currently in place on the network. To avoid
+causing unintended issues, the following limitations are proposed to be in
+place:
+
+| max | value |
+| --- | --- |
+| max transaction size | 4096 |
+| max num signatures per transaction | 42 |
+| max num accounts | 64 |
+| max num instructions | 64 |
+| max accounts/instruction | 255 |
+
+The above limits are useful to spec, but not necessarily different than what is
+currently in place on the network. Any transaction violating these constraints
+will be considered invalid and will not be included in the chain. Violations are
+considered sanitization failures.
+
+While the above lists sanitization failures, we will keep the current runtime 
+max limit of 12 signatures per transaction.
+
+### TransactionConfigMask
+
+The transaction config mask is used to configure specific fee and resource 
+requests in a transaction.
+It is intended to be flexible enough to add new fields in the future, and is 
+initially aimed at replacing the ComputeBudgetProgram instructions that are 
+currently used to configure transactions.
+Each bit in the mask represents 4-bytes in the `ConfigRequests` array.
+If a configured value, such as priority-fee, needs more than 4-bytes a field can
+use 2 bits in the mask.
+
+Initially supported fields and assigned bits:
+
+- [0, 1] - priority-fee. 8-byte LE u64.
+- [2] - compute-unit-limit. 4 byte LE u32.
+- [3] - requested loaded accounts data size limit. 4 byte LE u32.
+- [4] - requested heap size. 4 byte LE u32.
+
+For 2 bit fields, such as priority-fee, both bits MUST be set. If only one of 
+the bits is set, the transaction is invalid and cannot be included in blocks.
+
+For TxV1 transactions, any ComputeBudgetProgram instructions are ignored for 
+configuration, even if they are invalid.
+The instructions will still consume compute-units if included.
```

Comment:
Maybe say that they will still be processed as successful no-op instructions that consume compute units if included?